overlord/managers_test.go

```diff
@@ -113,6 +115,16 @@ func (ms *mgrsSuite) SetUpTest(c *C) {
 	err := os.MkdirAll(filepath.Dir(dirs.SnapStateFile), 0755)
 	c.Assert(err, IsNil)
 
+	// mock snap-confine profile inside core snap rev 1 and 30, required by setup-profiles tasks.
```

Comment:
Why 1-30 ? Do we really use so many mocked snap-confines?

Reply:
No, we use only 2  revisions: "1" and "30". It's not a 1..30 range :)

Reply:
Uhhh - silly me - sorry for the noise.